packages/documentation-site/patternfly-docs/content/design-guidelines/components/progress-stepper/progress-stepper.md

```diff
@@ -34,6 +34,12 @@ Use a progress stepper to convey to a user how many steps are required to comple
 
 <img src="./img/progress-stepper-table-view-popover.png" alt="Image showing example of progress stepper in a table view with a popover." width="1600" />
 
+### When to use progress stepper vs. wizard
+
+Use a progress stepper to indicate to the user where they are in a step-by-step linear process on a single page. The steps in the process could be completed by the user or happening in the background without the user completing the task themselves. 
```

Comment:
I would maybe try to break this up into bullets to match how we've done it in other guidelines -

Use a progress stepper:
- To indicate to the user where they are in a step-by-step linear process on a single page
- When the steps are happening in the background without the user completing the task themselves

Use a wizard:
- To guide a user through a multi-step flow
- When the steps are complex enough to need multiple pages 


I would also maybe add something about wizards taking the user out of context, and progress keeping them in the same place? Technically, the wizard keeps someone on the same page, and the progress stepper is still a multiple step flow except the content below changes instead of fully flipping to the same page. Just thinking out loud...

Reply:
Just changed the formatting! I tried to update the content a bit but I'm not sure I'm understanding correctly, so it might need another pass!